tests/functional/z_cluster/nodes/test_nodes_maintenance.py

```
@@ -610,3 +625,276 @@ def test_pdb_check_simultaneous_node_drains(
+    @pytest.fixture(autouse=True)
+    def setup(self, request, nodes):
+        """
+        Initialize sanity helpers and register a finalizer to restart
+        any nodes that were stopped during the test.
+
+        """
+        self.sanity_helpers = Sanity()
+        self.stopped_node_objs = []
+        self._nodes = nodes
+
+        def restart_stopped_nodes():
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `restart_stopped_nodes`

Add return type annotation: `None`

(ANN202)